src/create-post.html

```diff
@@ -0,0 +1,121 @@
+<!DOCTYPE html>
+<html lang="en">
+
+<head>
+  <meta charset="UTF-8">
+  <meta http-equiv="X-UA-Compatible" content="IE=edge">
+  <meta name="viewport" content="width=device-width, initial-scale=1.0">
+  <link rel="stylesheet" type="text/css" href="./dist/css/create.css">
+
+  <!--font awesome link-->
+  <script src="https://kit.fontawesome.com/03d5a1f99a.js" crossorigin="anonymous"></script>
+
+  <title>Inbound Media | Create Post</title>
+
+
+  <script defer type="module" src="./routes/create-post.js"></script>
+</head>
+
+<body>
+
+  <header>
+    <nav class="navbar bg-white">
+      <h1>
+        <a href="#"><i class="fa-brands fa-facebook"></i>Inbound Media</a>
+
+      </h1>
+      <ul>
+        <!--<li> <a href="posts.html">Post</a></li>
+          <li> <a href="edit.html">Edit</a></li>-->
+        <li> <a href="create-post.html">Create Post</a></li>
+        <li> <a href="single-post.html">Single Post</a></li>
+        <li> <a href="register.html">Register</a></li>
+      </ul>
+    </nav>
+  </header>
+  <!--NAV FINISH-->
+
+  <section class="landing">
+    <div class="dark-overlay">
+      <div class="landing-inner">
+        <h1 class="x-large">Welcome create a posts and share your ideas others!</h1>
+      </div>
+
+      <div class="container_bg">
+
+        <form action="#" class="form-signin">
+          <h2>Create post</h2>
+          <div id="flash"></div>
+          <div class="form-group">
+            <label for="email">Title</label>
+            <input id="title" name="title" required type="text" class="form-control" placeholder="Title" />
+
+
+          </div>
+
+          <div class="form-group">
+            <label for="username">Body</label>
+            <input id="body" name="body" required type="text" class="form-control" title="Must Container 4 Characters"
+              placeholder="Body" />
+
+          </div>
+
+          <div class="form-group">
+            <label for="tags">Tags</label>
+            <input id="tags" name="Tags" required type="text" class="form-control" placeholder="Tags" />
+
+          </div>
+
+          <div class="form-group">
+            <label for="media">Media</label>
+            <input id="media" name="Media" required type="text" class="form-control"
+              placeholder="Enter a image link for Banner" />
+          </div>
+          <button id="create-submit" type="button" class="btn btn-primary">Create Entry</button>
```

Comment:
```suggestion
          <button id="create-submit" class="btn btn-primary">Create Entry</button>
```

Type button doesn't trigger a submit, have you tested this?

Reply:
Yes I have tested it and it works with the type it doesn´t trigger I have Tried your suggestion, but then I couldn't log in.